frontend/javascripts/oxalis/view/components/setting_input_views.tsx

```diff
@@ -487,7 +487,7 @@ export class UserBoundingBoxInput extends React.PureComponent<UserBoundingBoxInp
               }
             >
               <DeleteOutlined
-                onClick={onDelete}
+                onClick={allowUpdate ? onDelete : () => {}}
                 style={allowUpdate ? iconStyle : disabledIconStyle}
                 disabled={!allowUpdate}
```

Comment:
The `disabled` prop doesn't seem to be supported for icons which explains this behavior. You can test this by simply setting `disabled` regardless of `allowUpdate` - nothing changes. To avoid further confusion, please remove the attribute.